posts/month-in-mathlib-oct-2021.md

```diff
@@ -0,0 +1,70 @@
+* [PR #9461](https://github.com/leanprover-community/mathlib/pull/9461) :: feat(measure_theory/covering/besicovitch_vector_space): vector spaces satisfy the assumption of Besicovitch covering theorem
+* [PR #9576](https://github.com/leanprover-community/mathlib/pull/9576) :: feat(measure_theory/covering/besicovitch): the measurable Besicovitch covering theorem
+* [PR #9679](https://github.com/leanprover-community/mathlib/pull/9679) :: feat(measure_theory/covering/besicovitch): remove measurability assumptions
+* [PR #9680](https://github.com/leanprover-community/mathlib/pull/9680) :: feat(measure_theory/covering/vitali): Vitali covering theorems
+
+* [PR #8976](https://github.com/leanprover-community/mathlib/pull/8976) :: feat(order/jordan_holder): Jordan Hölder theorem
+
+* [PR #9252](https://github.com/leanprover-community/mathlib/pull/9252) :: feat(topology/homotopy): add `homotopy_with`
+* [PR #9141](https://github.com/leanprover-community/mathlib/pull/9141) :: feat(topology/homotopy/path): Add homotopy between paths
+
+* After a long time of inactivity, some foundational parts of the perfectoid project were moved to mathlib.
+  - [PR #9521](https://github.com/leanprover-community/mathlib/pull/9521) adic topology
+  - [PR #9589](https://github.com/leanprover-community/mathlib/pull/9589) valued fields
+
+* [PR #9646](https://github.com/leanprover-community/mathlib/pull/9646) :: feat(number_theory/liouville): Liouville numbers form a dense Gδ set
+* [PR #9702](https://github.com/leanprover-community/mathlib/pull/9702) :: feat(number_theory/liouville): the set of Liouville numbers has measure zero
+
+* [PR #9496](https://github.com/leanprover-community/mathlib/pull/9496) :: feat(analysis/box_integral): Divergence thm for a Henstock-style integral
+* [PR #9811](https://github.com/leanprover-community/mathlib/pull/9811) :: feat(measure_theory/integral): Divergence theorem for Bochner integral
+
+* [PR #9817](https://github.com/leanprover-community/mathlib/pull/9817) and [PR #9834](https://github.com/leanprover-community/mathlib/pull/9834)
+  filled a glaring hole in the field theory library: finite fields of the same cardinality are isomorphic.
+
+* [PR #9540](https://github.com/leanprover-community/mathlib/pull/9540) :: feat(analysis/normed_space/banach): open mapping theorem for maps between affine spaces
+* [PR #9440](https://github.com/leanprover-community/mathlib/pull/9440) :: feat(analysis/asymptotics): Define superpolynomial decay of a function
+* [PR #9662](https://github.com/leanprover-community/mathlib/pull/9662) :: feat(group_theory/sylow): Frattini's Argument
+* [PR #7825](https://github.com/leanprover-community/mathlib/pull/7825) :: feat(combinatorics/hall): generalized Hall's Marriage Theorem
+* [PR #9924](https://github.com/leanprover-community/mathlib/pull/9924) :: feat(analysis/inner_product_space/dual): complex Riesz representation theorem
```

Comment:
Possible commentary: Frédéric Dupuis proved the Riesz representation theorem in Lean over both `ℝ` and `ℂ` (see  #4379), but, since mathlib lacked conjugate-linear maps at that time, the `ℂ`-version was stated in a rather awkward way.  Conjugate-linear maps were added to mathlib [last month](https://leanprover-community.github.io/blog/posts/month-in-mathlib-sep-2021/) (that's a link the September blog post) and now the `ℝ` and `ℂ` versions of the theorem can be stated uniformly.

Reply:
Thanks, done.